### ISSUE TITLE
Question : upgrading versions

### ISSUE THREAD

**Issue by tgtje** [e107inc/e107#613]
What would be considered as MINIMAL existing version to eventually upgrade to v2 ?
Due to many changes which have been on versions prior to v1 (starting at v 0.7.24 and upwards).

Although there will still be users who did never upgrade up to version 1.0.4 (which should have been made overall), it is probably expected that present files will not accommodate certain settings in (guessing) pre v 0.7.26 versions.

So basically : what is the pre-requisite (# required or necessary as a prior condition) before an update is entended ?
